runtime/dancebox/src/lib.rs

```diff
@@ -702,12 +706,6 @@ impl_runtime_apis! {
             let mut list = Vec::<BenchmarkList>::new();
 
             list_benchmark!(list, extra, frame_system, SystemBench::<Runtime>);
-            list_benchmark!(
```

Comment:
Why was this deleted?

Reply:
It was wrong, notice it says "PalletConfigurationBench" when the name is "tp_author_noting_inherent", I noticed it when running the benchmark scripts because the config benchmarks were there twice.